R/prepare_pacta_dashboard_data.R

```diff
@@ -342,7 +342,8 @@ prep_key_bars_portfolio(
 
 
 # put JSON and CSV outputs into a zip archive ----------------------------------
-zip_outputs(output_dir)
+
+# zip_outputs(output_dir)
```

Comment:
Can you explain why you are disabling this? 
I know from a call we had that it was causing an error that it couldn't recover from, but can you please specify what the error was so that we have an idea of how to solve it

Reply:
We do need this functionality, so I'd prefer we actually just address this now, rather than disable it

Reply:
I don't have enough information to know quite what's wrong.

Full Docker run logs:
```
Digest: sha256:7b5a43eab6cedcd5fcb4c64d0b14d8e3e30b4b107bd73d9b2eb1560384254b7d
Status: Downloaded newer image for ghcr.io/rmi-pacta/workflow.pacta.dashboard:pr-35
Error in write_delim(x, file, delim = ",", na = na, append = append, col_names = col_names,  : 
  is.data.frame(x) is not TRUE
Calls: <Anonymous> ... zip_outputs -> %>% -> <Anonymous> -> write_delim -> stopifnot
Execution halted
```

https://github.com/RMI-PACTA/workflow.pacta.dashboard/actions/runs/12234692851/job/34124506400#step:7:93

Reply:
I can address the functionality in a separate PR, once I've build up some telemetry around it (probably logging file names)